index.html

```diff
@@ -0,0 +1,129 @@
+      <nav class="heading">
+        <img class="heading-elements" src="images/facebook.jpg" width="24" height="24" alt="facebook">
+        <img class="heading-elements" src="images/twitter.png" width="24" height="24" alt="twitter">
+        <span class="text-white fw-bold heading-elements">English</span>
+        <span class="text-white fw-bold heading-elements">My Page</span>
+        <span class="text-white fw-bold heading-elements">Logout</span>
+      </nav>
+      <div class="container bottomNav">
+        <!-- Brand Logo -->
+        <a class="nav-desktop navbar-brand" href="#"><img src="images/cc-logo.jpg" width="48" alt="logo"></a>
+        <!-- Contents -->
+        <a class="nav-desktop text-decoration-none" id="content-cc" href="#contents">Contents</a>
+        <!-- Coding Challenges -->
+        <a class="nav-desktop text-decoration-none" id="challenge-cc" href="#coding-challenges">Coding Challenges</a>
+        <!-- About -->
+        <a class="nav-desktop text-decoration-none" id="about-cc" href="about.html">About CC</a>
+        <!-- Hamburger menu -->
+        <img id="hamburger" src="images/bars-solid.jpg" width="36" height="48" alt="menu-icon">
+      </div>
+    </nav>
+    
+    <!-- Navigation Menu -->
+    <div class="close-menu" id="navMenu">
+        <img id="close" src="images/x-solid.svg" width="24"  alt="close">
+        <ul id="nav-list" class="text-decoration-none list-unstyled ms-auto">
+            <li class="mx-auto">
+                <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="mx-auto">
+                <a class="nav-link" href="about.html">About</a>
+            </li>
+        </ul>
+    </div>
+
+    <section class="container-fluid home-page">
+        <h3 class="motto">"Preparing you for the language of the future!"</h3>
+        <h1 class="description-header">CODING CHALLENGES</h1>
+    </section>
+    <div id="describer">
+        <p id="description">Wanna challenge yourself with interesting programming tasks? Get started now by proceeding to the 'Coding Challenges' section of this page.</p>
+    </div>
+    <div>
+      <h2 class="date cocogoose">2015.10.15(THU) ~ 16(FRI)</h2>
+      <p class="loc lato">@National Museum of Korea. Art Center Nabi and more</p>
+    </div>
+  </div>
+  <section id="contents" class="contents d-flex gap-4">
+    <h1 class="content-header">Contents</h1>
+    <div class="tbl d-flex flex-md-row align-items-center gap-3">
+      <div class="row flex-lg-column row1"> 
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/book.svg" width="48" height="48" alt="book">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Reading Materials
+        </div>
+        <div class="col-6 col-md-12">
+          Offer references for reading materials to give context to your studies and to allow you to dig deep on your lessons.
+        </div>
+      </div>
```

Comment:
### Your Cards
![one-big-card (copy)](https://user-images.githubusercontent.com/93039359/181394636-fe231d72-2d72-4442-a561-04a6745126ee.png)

- [ ] First card is bigger than the other cards. Please adjust its height and width to look similar to other cards. :blush:
- [ ] It should show a border on hovering on it. Please fix this issue as requested by previous reviewer :+1: